bin/p2-preparer/main.go

```diff
@@ -34,6 +34,12 @@ func main() {
 		logger.WithError(err).Fatalln("invalid parameter")
 	}
 
+	if preparerConfig.ReadyFile != "" {
+		if _, err := os.Stat(preparerConfig.ReadyFile); os.IsNotExist(err) {
+			logger.WithError(err).Fatalln("ready file not present")
```

Comment:
yes, if I am understanding the design doc correctly, it fails if the ready file is not present